### ISSUE TITLE
consider hosting documentation using readthedocs 

### ISSUE THREAD

**Issue by shahzebsiddiqui** [E4S-Project/E4S-Project.github.io#4]
It may make your life easier if you consider hosting docs on https://readthedocs.org/ that way you dont have to write html pages. You can still write pages in RST or Markdown and readthedocs will host your site. Just create an account https://readthedocs.org/accounts/signup/ and you can see the readthedocs guide https://docs.readthedocs.io/en/stable/index.html 

I have used Sphinx for building your docs, it fairly simple to setup and used by many open-source projects.

**Comment by eugeneswalker** [comment]
Hi Shazeb, thank you for the suggestion. I have shared this with our team. We agree this looks promising. After our call earlier today, and after reviewing the other issues you raised, we are going to update our existing documentation in order to eliminate confusion as quickly as possible, and may make the change to Sphinx+Readthedocs down the road.

**Comment by shahzebsiddiqui** [comment]
That's great to hear @eugeneswalker. I look forward to the new docs. Yea i will try to provide more feedback, the documentation is going to be super important if you want folks to start using e4s, otherwise they will be turned off. Once the new site is up i will provide some more feedback and try out e4s myself.

**Comment by maherou** [comment]
@shahzebsiddiqui, I agree we can do a better job with documentation.  We have some efforts underway.  It is less clear to me that readthedocs is the best choice anymore.  gh-pages seems to be another viable approach.  It is particularly attractive since gh-pages is within the same ecosystem.  Interested in your thoughts.

**Comment by maherou** [comment]
Yes, it does.  Our documentation plan for E4S is to provide mostly basic documentation for E4S itself.  For the products that are part of E4S, we are working on a documentation portal that would rake high-level information from the individual product pages, refreshing on a regular cadence (e.g., daily), but redirect to the individual product documentation (e.g., PETSc website) for more details.
